servicesCore/requestCore_service.py

```diff
@@ -1,35 +1,219 @@
+                    # maybe a creation
+                    event = self._web3Single.decodeTransactionLog(self._abiRequestCore, 'Created', txReceipt.logs[0])
+                    if event:
+                        request = self.getRequest(event.requestId)
+            else:
+                # if not mined
+                methodGenerated = ccyContractservice.generateWeb3Method(transaction.method.name, self._web3Single.resultToArray(transaction.method.parameters))
+                options = {
+                    'from': transaction.from,
+                    'gas': transaction.gas,
+                    'value': transaction.value
+                }
+
+                try:
+                    test = await self._web3Single.callMethod(methodGenerated, options)
+                except Exception as e:
+                    warnings.append('transaction may failed: ' + e)
+
+                if transaction.gasPrice < config.ethereum.gasPriceMinimumCriticalInWei:
+                    warnings.append('transaction gasPrice is low')
+
+            errors = None if not errors else errors
+            warnings = None if not warnings else warnings
+            return (request, transaction, errors, warnings)
+
+        except Exception as e:
+            raise e
 
-    def getRequestByTransactionHash(self, hash:str):
-        pass
 
     def getRequestEvents(self, requestId: str, fromBlock: int = None, toBlock: int = None):
         pass
 
-    def getRequestsByAddress(self, address: str, fromBlock: int = None, toBlock: int = None):
-        pass
+    async def getRequestsByAddress(self, address: str, fromBlock: int = None, toBlock: int = None):
+        """
+        Get the list of requests connected to an address
+        :param address: address to get the requests
+        :param fromBlock: search requests from this block
+        :param toBlock:  search requests until this block
+        """
+        try:
+            networkName = self._web3Single.networkName
+
+            # get events Created with payee == address
+            eventsCorePayee = await self._instanceRequestCore.getPastEvents('Created', {
+                'filter': {'payee': address},
+                'fromBlock': fromBlock if fromBlock else requestCoreArtifact['networks'][networkName]['blockNumber'],
+                'toBlock': toBlock if toBlock else 'latest'
+            })
+
+            # get events Created with payer == address
+            eventsCorePayer = await this._instanceRequestCore.getPastEvents('Created', {
+                'filter': {'payer': address},
+                'fromBlock': fromBlock if fromBlock else requestCoreArtifact['network'][networkName]['blockNumber'],
+                'toBlock': toBlock if toBlock else 'latest'
+            })
+
+            # clean the data and get timestamp for request as payee
+            # TODO: implement this, I believe it is just adding 
```

Comment:
If this isn't done can you have this raise a `NotImplemented` error so that people don't call it accidentally. Also maybe open an issue for it so that people don't scroll by and see that it's not `pass` and assume that it's done.

Reply:
will do